tools/monorepo-utils/src/core/environment.ts

```diff
@@ -14,3 +14,7 @@ export const getEnvVar = ( varName: string, isRequired = false ) => {
 
 	return value || '';
 };
+
+export const isGithubCI = () => {
+	return process.env.GITHUB_ACTIONS === 'true';
```

Comment:
Added a simple sync way to check if env is CI, useful for deciding how to log etc.

Reply:
@psealock oh nice, it looks like they added the CI variable to GH but I didn't realize (I don't remember seeing that in the docs) I don't care too much about the implementation but yeah maybe its nice for this to be a function just so you can quickly know to reach for that functionality?

I'm easy 😄